pontoon/base/models.py

```diff
@@ -1148,6 +1158,9 @@ def for_project_locale(self, project, locale, paths=None, filter_type=None,
             elif filter_type == 'unchanged':
                 entities = self.objects.unchanged(locale)
 
+            elif filter_type in Translation.authors(locale, project, paths).values_list('email', flat=True):
```

Comment:
What would you say to change this DB-call for the regexp email matching? Could be a little bit faster.